docs/eos-learn/blockchain-security.md

```diff
@@ -0,0 +1,8 @@
+---
+id: blockchain-security
+title: Blockchain Security
+sidebar_label: Security
+---
+## Exploding vulnerabilities in Blockchain
```

Comment:
@hanakoori01 I think the word you are looking for is "exploiting"

Reply:
@hanakoori01 it would also be great to get an intro text for this video 

Maybe @kuronosec can help us come up with a short text